lib/system.nim

```diff
@@ -258,6 +258,12 @@ else:
     UncheckedArray*[T]{.unchecked.} = array[0,T]
     ## Array with no bounds checking.
 
+template toCstring*(a: ptr char): cstring = cast[cstring](a)
+template toCstring*[N](a: ptr array[N, char]): cstring = cast[cstring](a)
+template toCstring*[N](a: array[N, char]): cstring = cast[cstring](a.addr)
+template toCstring*[N](a: ptr UncheckedArray[char]): cstring = cast[cstring](a)
+template toCstring*(a: UncheckedArray[char]): cstring = cast[cstring](a.addr)
```

Comment:
Remove all of this. Do not add new stuff to system.nim unless it is required. And if it is required, justify why it needs to be in system.nim.